syncd/tests.cpp

```diff
@@ -718,145 +718,6 @@ void test_invoke_dump()
     assert(mockCallArg == SAI_FAILURE_DUMP_SCRIPT);
 }
```

Comment:
Should `test_invoke_dump` be also moved to `syncd/tests/TestSyncdBrcm.cpp`, or remain in this file?

Reply:
@saiarcot895 ideally - yes